src/toil/test/wdl/builtinTest.py

```diff
@@ -230,6 +246,22 @@ def testFn_Length(self):
         self.assertEqual(3, length(['a', 'b', 'c']))
         self.assertEqual(0, length([]))
 
+    def testFn_Zip(self):
+        """Test the wdl built-in functional equivalent of 'zip()'."""
+        xs = [1, 2, 3]
+        ys = ['a', 'b', 'c']
+        zipped = wdl_zip(xs, ys)  # [WDLPair(1, 'a'), WDLPair(2, 'b'), WDLPair(3, 'c')]
+
+        self.assertEqual(3, len(zipped))
+
+        for index, val in enumerate(xs):
```

Comment:
Nit: it might be clearer to enumerate `zipped`:

```
        for index, val in enumerate(zipped):
            self.assertIsInstance(val, WDLPair)
            # check left and right values
            self.assertEqual(xs[index], val.left)
            self.assertEqual(ys[index], val.right)
```